### ISSUE TITLE
What am I supposed to run?

### ISSUE THREAD

**Issue by jaredbrinkley160** [caslabuiowa/BeBOT#16]
When I run the example files I constantly get ModuleNotFoundError. I am in the anaconda python terminal, my directory is correct (BeBOT-master) and I have completed all of the setup instructions in readme. I have been doing, for example, python 2_1Examples.py and get ModuleNotFoundError: No module named 'polynomial' 

I am new to Github, so I am sure there is something simple I am not understanding, but I'd appreciate some help.
![Bebot_error](https://github.com/caslabuiowa/BeBOT/assets/171302107/1ad24bae-1467-4341-93bf-897de62da0df)

**Comment by magicbycalvin** [comment]
TL;DR; 
Add both the BeBOT directory and its parent directory to the PYTHONPATH.

No worries, I completely understand the challenge! 
For now, the band-aid solution is to run spyder and then add both the BeBOT folder AND the folder that contains it to the Python path. I am currently using spyder 5.5.1 and if you are using a similar version, you should have the option to modify the PYTHONPATH by clicking the yellow and blue python icon at the top of the screen to add paths. In your case, you should add both ` C:\Users\jared\Downloads` AND ` C:\Users\jared\Downloads\BeBOT-master`. From there, you should be able to run the file directly within Spyder. From your screenshot, it looks like you may be on Windows. I have not tested this fix on Windows, but I am assuming it should work fine. Let me know if you run into issues though and I'll see what I can do to fix it.

In the meantime, I'll do some additional work and get this working for future releases.

**Comment by jaredbrinkley160** [comment]
That worked, much appreciated!
